### PR TITLE
Add option to hide table header

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: #139

### CODE REVIEW

File: src/utils/test.ts [hunk 1]
```diff
@@ -248,6 +248,7 @@ export const createTableRequestConfig = (item: Partial<TableRequestConfig>): Tab
  */
 export const createTableConfig = (table: Partial<TableConfig>): TableConfig => ({
   name: '',
+  showHeader: true,
```
Comment (asimonok) [inline src/utils/test.ts:251]: all options in test utils should be disabled to make test data is more clear and predictable due to necessary to enable particular options per test

**Review by asimonok — CHANGES_REQUESTED** [review]

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

**Review by mikhail-vl — APPROVED** [review]
Looks great!

**Pull request merged**